src/Layer/LabelLayer.js

```diff
@@ -34,6 +152,9 @@ class LabelLayer extends Layer {
      * contains three elements `name, protocol, extent`, these elements will be
      * available using `layer.name` or something else depending on the property
      * name.
+     * @param {boolean} [config.performance=false] - remove labels that have no chance of being visible.
```

Comment:
If I understand correctly, if set to true, we remove culled out labels. Why don't we do that in all cases?

Reply:
yes, it is true that you can set the default to true

Reply:
Would there be a point for user not to use `config.performance=true` ? What would be the use case of having labels inside a `LabelsNode` that will never be displayed ? Maybe `performance = true` should be the only behavior (without letting user parametise it) ?